system/keyboardlayouts/chinese.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+Please use English language names instead.
+Default font lacks support for all characters
+-->
+<keyboardlayouts>
+  <layout language="Chinese" layout="BasePY" codingtable="BasePY">
```

Comment:
What's the `layout` attribute for? It's value is identical to the `codingtable` attribute and does not seem to be used in the code. Furthermore having a `layout` attribute in a `<layout>` tag is rather odd.